radixdlt/babylon-wallet-android
# Transaction signing & ROLA (Batch 4)

## Description
* The wrapped `PublicKey`, `Signature`, `SignatureWithPublicKey` from #813 are now replaced with the corresponding ones from Sargon.
* The `PrivateKey` implementation now interacts with Sargon's objects.
* Transaction notarisation now works with Sargon's objects. 
* Rola works with the new objects. 
* A new block in settings (only in debug mode when `EXPERIMENTAL_FEATURES_ENABLED`) shows the dependencies in Sargon, RET, Scrypto and Signaling Server

## How to test
* Test with some transactions
* Test faucet
* Test ROLA auth
* After a transaction is completed and you see the success dialog, click/long-click on the `txid_...` address to see that it works.

## PR submission checklist
